### DISCUSSION TITLE
Getting a pokemon's previous generation stats

### DISCUSSION THREAD

**Discussion by diogofd8** [PokeAPI/pokeapi#891] [Q&A] [answered]
Hello!
Do know how to query pokeAPI for the gen1 special stat? 

I know the pokemon stats are listed in v2/pokemon/{pokemon_id}/stats/ but that doesn't seem to account for the pre special split era or even the small stat changes in some pokemon such as pidgeot.

`https://pokeapi.co/api/v2/pokemon/pidgeot`
```
"stats":[{"base_stat":83,"effort":0,"stat":{"name":"hp","url":"https://pokeapi.co/api/v2/stat/1/"}},{"base_stat":80,"effort":0,"stat":{"name":"attack","url":"https://pokeapi.co/api/v2/stat/2/"}},{"base_stat":75,"effort":0,"stat":{"name":"defense","url":"https://pokeapi.co/api/v2/stat/3/"}},{"base_stat":70,"effort":0,"stat":{"name":"special-attack","url":"https://pokeapi.co/api/v2/stat/4/"}},{"base_stat":70,"effort":0,"stat":{"name":"special-defense","url":"https://pokeapi.co/api/v2/stat/5/"}},{"base_stat":101,"effort":3,"stat":{"name":"speed","url":"https://pokeapi.co/api/v2/stat/6/"}}]
```

According to Bulbapedia, Pidgeot had 70 Special in Gen 1, along with 91 Speed from Gen 1-5, getting 101 Speed from Gen 6 onward.

**Comment by Naramsim** [comment, marked as answer]
Hi, yes, currently we lack past stats. We are on the latest gen. 

We have some changelog tables but for `stats` there's none. So you won't find this info in our API currently.
